grapher/Models/Mouse/MouseWatcher.cs

```diff
@@ -740,15 +735,36 @@ public void ReadMouseMove(Message message)
 
             outSize = GetRawInputData((IntPtr)message.LParam, RawInputCommand.Input, out rawInput, ref size, Marshal.SizeOf(typeof(RAWINPUTHEADER)));
 
-            if (PollRateField.Data != PollRate)
+            if (SettingsManager.PollRateField.Data != PollRate)
             {
-                PollRate = PollRateField.Data;
-                PollTime = 1000 / PollRate;
+                PollRate = SettingsManager.PollRateField.Data;
+                PollTimeRecip = Math.Abs(SettingsManager.PollRateField.Data) / 1000;
```

Comment:
I'd put this in the setter for PollRate so that whenever PollRate is updated PollTimeRecip is as well